config/locales/en.yml

```diff
@@ -10,7 +10,12 @@ en:
       hint_stub_creation_html: <span>Hint:</span>To see your component make sure you've created stubs
       tip_stub_creation_html: 'You should write stub examples here:'
     index:
-      description: Select one of the components from the side to view its examples and documentation.
+      description_html:
+        <p>Select one of the components from the side to view its examples and documentation.</p>
+        <br />
+        <br />
+        <p>To replace the content on this page, add the directory `app/views/mountain_view` in your application,
```

Comment:
Should `app/views/mountain_view` be inside `<code>` tags instead of backticks (`)?

Reply:
Both are no win code tags, with no attributes.